chat-plugins/scavengers.js

```diff
@@ -26,6 +28,42 @@ if (scavengersRoom) {
 	}
 }
 
+let scavengersLB;
+let dataFile = './config/chat-plugins/scavdata.json';
+fs.readFile(dataFile, 'utf8', function (err, contents) {
+	if (err) {
+		scavengersLB = {};
+		return;
+	}
+	scavengersLB = JSON.parse(contents);
+});
+
+function addPoints(username, points) {
+	let id = toId(username);
```

Comment:
probably should check there's an ID left.   If someone uses addPoints on ``( ͡° ͜ʖ ͡°)`` it could be interesting....